.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+ - 1.9.3
```

Comment:
Yeah, I don't think we need it. Does `jruby-19mode` means ruby 1.9.x? I mean should we drop it as well?